x-pack/plugins/security/common/licensing/license_features.ts

```diff
@@ -46,7 +51,7 @@ export interface SecurityLicenseFeatures {
   /**
    * Describes the layout of the login form if it's displayed.
    */
-  readonly layout?: string;
+  readonly layout?: LoginLayout;
```

Comment:
> I don't love that this lives here, it (and linksMessage below) strikes me as separate from license features

Me neither, it's a legacy that we should get rid of at some point.

> Also: it looks like linksMessage maybe isn't needed anymore.
Hmm, will make sure it's not needed and remove it, thanks!

Reply:
Okay, found where we used `linksMessage` before: https://github.com/elastic/kibana/blob/7.6/x-pack/legacy/plugins/security/public/views/management/management.js#L30-L32

Previously we displayed this message when user was trying to access Security management views, but  Security wasn't available. With the new Management API we can't do this anymore since we can't know when user tried to access these views if Security management section is disabled since Management App itself handles that. Will remove this property.